inc/saivlan.h

```diff
@@ -160,13 +160,13 @@ typedef enum _sai_vlan_member_attr_t
     SAI_VLAN_MEMBER_ATTR_VLAN_ID = SAI_VLAN_MEMBER_ATTR_START,
 
     /**
-     * @brief Logical port ID
+     * @brief Bridge port ID. Valid only for .1Q Bridge ports
      *
      * @type sai_object_id_t
-     * @objects SAI_OBJECT_TYPE_PORT
+     * @objects SAI_OBJECT_TYPE_BRIDGE_PORT
```

Comment:
the Vlan object is valid only for .1Q bridge ,  and used vlan base bridge configuration (port vlan member, learning ...) 
.1D bridge are vlan unaware bridges